packages/scss/demo/components/pagination.html

```diff
@@ -0,0 +1,97 @@
+<h1 class="demo-divider" id="pagination">Pagination</h1>
+<section class="contentSection">
+    <h1>Pagination</h1>
+    <nav class="pagination" role="navigation" aria-labelledby="pagination-count">
+        <div id="pagination-count" class="pagination-count">
+            <span class="pagination-count-current">
+                <span class="u-mask">Results from</span>
+                1<span aria-hidden="true">-</span>
```

Comment:
You are missing a space between « 1» and « - » here. And you should replace your hyphen with a dash.